### PR TITLE
Assets page

### PR DESCRIPTION
**Pull request by tannerwelsh**
#### What does this PR do?

Adds an assets page to the Savile docs as a reference for asset files hosted in our S3 bucket on http://assets.turing.io/

#### Where should the reviewer start?
New `_data/assets.yml` file.

#### How should this be manually tested?
Confirm that all asset files on https://turing-savile-assets.s3.amazonaws.com/?list-type=2 are listed in the assets page.

#### Any background context you want to provide?
No

#### What are the relevant tickets?
[Add assets page and display all assets with URLs](https://3.basecamp.com/3494409/buckets/19192671/todos/3634997064)

#### Screenshots (if appropriate)
Screen Shot 2021-04-06 at 11 35 57 AM
Screen Shot 2021-04-06 at 11 36 00 AM

#### Any other deploy steps?
No

### CODE REVIEW

File: assets.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+layout: default
+permalink: /assets/
+---
+
+<p>Asset files used by Savile are hosted at <code>http://assets.turing.io/</code> and available for public use.</p>
+
+<p>To manage these assets, you will need an AWS account with permissions for the Savile Assets S3 bucket.</p>
+
+<h3 class="s-h3">Images</h3>
+<table class="assets-table">
```
Comment (ameseee) [inline assets.html:12]: I wonder if we can use the `spaced-table` and associated classes to make this table look a little more like the tables on other pages - every other row is dark/light ?

File: assets.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    {% assign images = site.data.assets | where: "group", "images" %}
+    {% for asset in images %}
+      <tr>
+        <td>
+          <img src="{{ asset.url }}" class="preview-image" />
+        </td>
+        <td>{{ asset.file }}</td>
+        <td>{{ asset.size }}</td>
+        <td>
+          <a href="{{ asset.url }}" target="_blank">{{ asset.url }}</a>
```
Comment (ameseee) [inline assets.html:31]: Should we use the `s-link` class here?

**Review by ameseee — COMMENTED** [review]
This is so awesome! I just left a couple comments on the table styling.

File: assets.html (hunk 1 above)
Comment (tannerwelsh) [reply assets.html:12]: Yes! Thanks for the reminder, that would be better

**Comment by tannerwelsh** [comment]
Thanks for the review @ameseee ! Here's what the tables look like now:

Screen Shot 2021-04-06 at 2 07 16 PM

**Pull request merged**